src/tests/Microsoft.PowerFx.Connectors.Tests/BasicRestTests.cs

```diff
@@ -72,7 +72,7 @@ public async void ValidateHttpCalls(int apiFileNumber, string fxQuery, string ht
             var checkResult = engine.Check(fxQuery, options: _optionsPost);
```

Comment:
These changes seem kind of unnecessary in tests. From what I can tell, it's only a concern in library code because we don't know if the caller will block.

Reply:
This is a general good practice and a (tiny) perf improvement but I agree it's not a drama if we would be missing those in tests.